packages/editor-ui/src/components/ParameterInputList.vue

```diff
@@ -258,7 +265,21 @@ export default mixins(
 </script>
 
 <style lang="scss">
-.paramter-input-list-wrapper {
+.parameter-input-list-wrapper {
+	display: flex;
+	flex-direction: column;
+	padding-top: var(--spacing-xs);
+
+	&.has-reordered-credentials-input {
```

Comment:
not a fan of this approach. we should think of a less-hacky global way to do this... would rather leave as is rather than this. 
like trying to find position automatically if possible or even adding a property flag `showCredentialsAfter: true`

Reply:
Refactored as `showCredentialsAfter` prop but open to suggestions if it's worth finding position automatically.

Reply:
let's attempt it... just time box it

Reply:
See [75eea89](https://github.com/n8n-io/n8n/pull/3222/commits/75eea89273b854110fa6d1f96c7c1d78dd3b0731)